CrashReporting/AddStackToCrashReports.md

```diff
@@ -65,10 +66,12 @@ That's fine. The browser should make a reasonable effort to collect call stack d
 An infinite loop in a worker doesn't cause the page or iframe to become unresponsive. This API reports only script on the main thread.
 
 ## When there are frames from multiple origins in a renderer, how do we ensure that stacks are attributed to the correct frame?
+
 To ensure that stacks are attributed to the correct frame, we need to send back the serialized frame token from the renderer to the browser along with the call stack. This way, the browser can verify that the call stack belongs to the same frame which the crash report is being generated for before attaching it to the report.
 
 ## If there is an extension executing scripts in the main world, how will you prevent the endpoint from knowing about the agent’s execution environment such as what extensions they have installed?
-Extension code injected into the main world may be visible.
+
+To address this, call stacks are automatically omitted whenever an extension’s content script is on the stack. By omitting the stack in these cases, we avoid exposing information about installed extensions in crash reports.
```

Comment:
I'm uneasy about just omitting leading to confusing states. An emptystring could lead to a lot of questions - Does the browser just not support the stacks? Does the browser think there was nothing on the stack? Was the header not set properly? etc.

Could we use a predictable string like "Stack Unavailable"? Or future-proofing, an enum that could contain reasons why the stack wasn't included? I could see "Extension", "User Opt-Out", and "Unknown". With "User Opt-Out" being something that might be an option for other UAs to choose to offer.

Reply:
Let's defer this, I can open an issue on crash reporting later.